code/modules/modular_computers/file_system/programs/jobmanagement.dm

```diff
@@ -1,3 +1,5 @@
+GLOBAL_VAR_INIT(time_last_changed_position, 0)
```

Comment:
What is this variable for?

Reply:
It used to be the ID console, and it had some behavior related the job management console, so I moved it there because I removed the ID console.

Reply:
Ah, just moved, alright.